opentelemetry/proto/metrics/v1/metrics.proto

```diff
@@ -337,6 +346,23 @@ enum DataPointFlags {
   // Bits 2-31 are reserved for future use.
 }
 
+// GaugeType defines valid gauge "sub-types", which are used to describe
+// gauges which have additional properties. These sub-types generally apply
+// to metrics which originate from protocols which support these additional
+// types, but are represented as gauges in OTLP.
+enum GaugeType {
+  // GAUGE_TYPE_NONE is the default GaugeType, which represents a gauge
+  // that does not have a sub_type. Since the OpenTelemetry API enforces types,
+  // this subtype is used for all metrics from OpenTelemetry instrumentation.
+  GAUGE_TYPE_NONE = 0;
```

Comment:
GAUGE_TYPE_NONE -> GAUGE_TYPE_STANDARD or something around this idea?

Reply:
I'm OK with that naming.  Agree it doesn't make sense to have a NONE, if it isn't a subtype anymore

Reply:
Updated.